pages/phases/phone-assessment-interviews/training.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: article
+permalink: /hiring-phases/phone-assessment-interviews/training/
+section: hiring-phases
+sidenav: hiring-phases
+phase: phone-assessment-interviews
+sub-phase-order: 011
+title: Training SMEs to Conduct Interviews
+intro: While phone interviews are being scheduled with applicants, facilitate a mandatory two-hour interview training for all SMEs who will conduct the next round of interviews that explains how to conduct structured interviews and assess an applicant’s proficiency with the required competencies based on their responses.
+---
+
+### Setting Up Phone Assessment Interview Training
+
+Send calendar invites to all SMEs who will conduct the next round of interviews for your hiring action. SMEs can participate in the phone interview training in person or remotely using a conference bridge line, depending on the means and norms at your agency.
+
+To prepare for your training:
+- Update the `[Phone Interview Training Presentation Template](toolkit-link)` with your agency-specific information, including both the questions for that interview round and final competencies and proficiencies from your job analysis workshop.  
```

Comment:
they need to do other steps as well:

1) prepare the interview template and attach it to the calendar  invite for the interview training. this is the primary document they will train to. in order to prepare the inteview template, download it from this site, insert all the appropriate questions, competencies, and proficiency levels. add the words "minimum qualification" next to the required proficiency level in the SME rating section. This may sit next to the "experienced" level or it could fit next to the  "master level"- use the job analysis workshop competency output to remember what the SMEs determined was required.

Reply:
2. Figure out how many SMEs passed resume review and break the SMEs up into round 1 and round 2 interviewers. make sure the SMEs will know the dates they need to book for their interviews and approximately how many interviews they need to conduct.  During the training you'll also want to provide the SMEs with a list of the applicant names so they can tell you which applicants they have a personal relationship with and therefore cannot interview. (refer to the instructions on our site about how to determine if you have a personal relationship)

Reply:
> add the words "minimum qualification" next to the required proficiency level in the SME rating section

Sorry, what is this? You said there's no indication of what the required proficiency level is anymore in the template.